### PR TITLE
Photo and Services tab

### PR DESCRIPTION
**Pull request by AlexStoskyi**

### CODE REVIEW

File: tests/PhotoTab/TC_409_VerifyServiceInputSection.spec.ts
Comment (AndreyCar) [inline tests/PhotoTab/TC_409_VerifyServiceInputSection.spec.ts:1]: I ran this test and got an error 
image

File: tests/PhotoTab/TC_594_VerifyMovingUploadedImage.spec.ts
Comment (AndreyCar) [inline tests/PhotoTab/TC_594_VerifyMovingUploadedImage.spec.ts:1]: I ran this test and got an error
image

File: tests/PhotoTab/TC_390_VerifyBackButton.spec.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import { faker } from '@faker-js/faker';
+import expectText from '../../helper/expectText.json';
+import { MainPage } from '../../pages/mainPage';
+
+test.beforeEach(async ({ page}) => {
+  await page.goto(url.create_unit);
+});
+
+test('TC_390_Verify ""Назад"" button', async ({ page }) => {
+  const loginPopUpPage = new LoginPopUpPage(page);
```
Comment (AndreyCar) [inline tests/PhotoTab/TC_390_VerifyBackButton.spec.ts:15]: Replace it with fixtures

**Review by AndreyCar — CHANGES_REQUESTED** [review]

File: tests/PhotoTab/TC_409_VerifyServiceInputSection.spec.ts
Comment (AlexStoskyi) [reply tests/PhotoTab/TC_409_VerifyServiceInputSection.spec.ts:1]: Screenshot 2024-10-23 at 17 53 29
дивно у мене інший тест впав))

Comment (AndreyCar) [reply tests/PhotoTab/TC_409_VerifyServiceInputSection.spec.ts:1]: @AlexStoskyi add the following to the playwright config
```ts
 expect: { timeout: 10_000 },
 use: {
    screenshot: 'only-on-failure',
    video: process.env.CI ? 'on-first-retry' : 'on',
    trace: 'on-first-retry',
  },
```

**Review by AndreyCar — APPROVED** [review]

**Pull request merged**